cibuildwheel/resources/install_certifi.py

```diff
@@ -0,0 +1,44 @@
+# Based on: https://github.com/python/cpython/blob/master/Mac/BuildScript/resources/install_certificates.command
+
+# install_certifi.py
+#
+# sample script to install or update a set of default Root Certificates
+# for the ssl module.  Uses the certificates provided by the certifi package:
+#       https://pypi.org/project/certifi/
+import os
+import os.path
+import ssl
+import stat
+import subprocess
+import sys
+STAT_0o775 = ( stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR
+             | stat.S_IRGRP | stat.S_IWGRP | stat.S_IXGRP
+             | stat.S_IROTH |                stat.S_IXOTH )
+
+if sys.version_info[0] == 2:
+    FileNotFoundError = OSError
+
+def main():
+    openssl_dir, openssl_cafile = os.path.split(
+        ssl.get_default_verify_paths().openssl_cafile)
+    print(" -- pip install --upgrade certifi")
+    subprocess.check_call([sys.executable,
+        "-E", "-s", "-m", "pip", "install", "--upgrade", "certifi"])
+    import certifi
```

Comment:
A few empty lines in this body of this function, to split off different logical parts, perhaps, if possible. For example here, to have a break between installing `certifi` and using it?